src/Fixer/Phpdoc/PhpdocTypesFixer.php

```diff
@@ -48,6 +48,7 @@
         'true',
         'void',
         '$this',
+        'scalar',
```

Comment:
shouldn't we extend tests for this fixer to check scalar as well ?

Reply:
Don't know. Because there are not all types tested as well. Just a small selection of em.

Reply:
We should probably add in some extra types that we missed, from https://github.com/phpDocumentor/TypeResolver/blob/master/src/TypeResolver.php#L47 too.

Reply:
Also, this list was sorted alphabetically, and now is not after adding `scalar` to it.

Reply:
Will add the missing types and resort them as well

Reply:
> Will add the missing types and resort them as well

I have already done this in another PR.

Reply:
Well then will remove the adding here to avoid conflicts